_rules/element-lang-matches-default-language-off6ek.md

```diff
@@ -325,11 +327,25 @@ The `lang` [attribute value][] of this `p` element is a [grandfathered tag][gran
 </html>
 ```
 
+#### Inapplicable Example 8
+
+The [text inheriting its programmatic language][] from this `div` element is only [whitespace][].
```

Comment:
I'm not following... the div is empty. It has no whitespace. Did something go wrong with Prettier or something?

Reply:
Yes, Prettier seems to remove them 😢 
Do you have an idea how to circumvent that?

(my current best would be to add some JS to do `foo.innerText = "    "` but that seems a bit too crazy…)